Controller/RoleController.php

```diff
@@ -89,13 +89,13 @@ public function viewRoleAction($roleId)
      */
     public function createRoleAction()
     {
-        if (!$this->isGranted(new Attribute('role', 'update'))) {
+        try {
+            $roleCreateStruct = new RoleCreateStruct(['identifier' => '__new__' . md5(microtime(true))]);
+            $role = $this->roleService->createRoleDraft($roleCreateStruct);
+        } catch (UnauthorizedException $e) {
```

Comment:
Why do you remove this change? `isGranted()` check was made on purpose, to clean up code (should check role/create though)